src/modules/add.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable import/no-duplicates */
+/* eslint-disable no-unused-vars */
+/* eslint-disable consistent-return */
+/* eslint-disable no-restricted-syntax */
```

Comment:
hello @Dipeshtwis  Thank you for reviewing my code.
I managed to decrease disabled linters from 16 to 6 and I really can't find a way to remove the others without affecting the code, is it okay if I submit with only 6 disabled or can you please give me a hint on how to fix the remaining errors without breaking the code.

Reply:
Hello @Dipeshtwis  the errors are:

1. Expected to return a value at the end of arrow function.
2. Exporting mutable 'let' binding, use 'const' instead  (although I need to use let because I am updating the array while removing )
3. Array.prototype.filter() expects a value to be returned at the end of arrow function
Thank you for the assistance, I have already pushed my code without disabling errors if you want to check it.

Reply:
For no. `2` you can disable the linter.